src/engine.rs

```diff
@@ -79,27 +75,19 @@ pub fn run(script_path: PathBuf, input_path: PathBuf) -> Result<FunctionRunResul
 #[cfg(test)]
 mod tests {
     use super::*;
+    use more_asserts::*;
     use std::path::Path;
 
     #[test]
-    fn test_runtime_under_threshold() {
+    fn test_runtime_duration() {
+        // This is using the https://github.com/Shopify/shopify-vm-prototype-script script
         let function_run_result = run(
             Path::new("tests/benchmarks/hello_world.wasm").to_path_buf(),
             Path::new("tests/benchmarks/hello_world.json").to_path_buf(),
         )
         .unwrap();
 
-        assert!(function_run_result.runtime <= Duration::from_millis(5));
-    }
-
-    #[test]
-    fn test_runtime_over_threshold() {
```

Comment:
I opted to remove this test because the `sleep` function is a no-op with the crate. I've adjusted the test above to ensure the `runtime` value is being set. Unfortunately we don't have a reliable way of making the `runtime` value more deterministic.

Reply:
Sounds good, but why is sleep a no-op?

Reply:
Thank you for this great explanation 😄